src/Utilities/CommandLineBuilder.cs

```diff
@@ -503,7 +503,7 @@ public void AppendFileNamesIfNotNull(ITaskItem[] fileItems, string delimiter)
 
         #region Appending switches with quoted parameters
 
-        internal static string FixCommandLineSwitch(string switchName)
+        public static string FixCommandLineSwitch(string switchName)
```

Comment:
It's used in one of the task tests and I could not easily make it internal visible. I am sure it could be done with some work, but did not seem to be worth it.